packages/utils/src/crypto/header/blinding.ts

```diff
@@ -0,0 +1,44 @@
+import { expand } from 'futoin-hkdf'
+import { PRIVATE_KEY_LENGTH, SECRET_LENGTH, HASH_ALGORITHM, HASH_LENGTH } from './constants'
+import { PRG_IV_LENGTH, PRG_KEY_LENGTH } from '../prg'
+import { PRP_IV_LENGTH, PRP_KEY_LENGTH } from '../prp'
+import type { PRGParameters } from '../prg'
+import type { PRPParameters } from '../prp'
+
+const HASH_KEY_BLINDING = 'HASH_KEY_BLINDING'
+const HASH_KEY_PRG = 'HASH_KEY_PRG'
+const HASH_KEY_PRP = 'HASH_KEY_PRP'
+
+export function deriveBlinding(secret: Uint8Array): Uint8Array {
```

Comment:
documentation?

Reply:
obsolete due to added method description